_posts/2023-07-24-whats-new-with-bzlmod.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: posts
+title: "What's New with Bzlmod?"
```

Comment:
nit: lower case "new"?

Reply:
hmm. it appears that we're not consistent w.r.t title case for blog post titles. (Even Wikipedia lists multiple house rules for title case: https://en.wikipedia.org/wiki/Title_case)

In this specific case, I agree that a lowercase "new" somehow looks better. So changed.